components/list/list-item-mixin.js

```diff
@@ -596,6 +630,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			'd2l-list-item-content-extend-separators': this._extendSeparators,
 			'd2l-dragging-over': this._draggingOver
 		};
+		const colorStyles = {
+			backgroundColor: this.color || undefined
```

Comment:
Jeff seems to be leaning towards not having validation and just not having the color render if it's invalid.

If we do want to validate but also want to allow any possible valid color value, after color is set we could then check the `background-color` value and see if it got wiped out and throw if so.